src/main/java/io/jenkins/plugins/gitlabbranchsource/LogCommentTrait.java

```diff
@@ -1,31 +1,43 @@
 package io.jenkins.plugins.gitlabbranchsource;
 
+import edu.umd.cs.findbugs.annotations.NonNull;
 import hudson.Extension;
+import javax.annotation.Nonnull;
 import jenkins.scm.api.SCMSource;
 import jenkins.scm.api.trait.SCMSourceContext;
 import jenkins.scm.api.trait.SCMSourceTrait;
 import jenkins.scm.api.trait.SCMSourceTraitDescriptor;
 import org.jenkinsci.Symbol;
 import org.kohsuke.stapler.DataBoundConstructor;
 
+
 public class LogCommentTrait extends SCMSourceTrait {
 
+    @NonNull
+    private String sudoUser;
+
     /**
      * Constructor for stapler.
      */
     @DataBoundConstructor
-    public LogCommentTrait() {
-        //empty
+    public LogCommentTrait(@Nonnull String sudoUser) {
```

Comment:
Please use `@DataBoundSetter` instead
I don't think it should be mandatory.

Reply:
Fixed in c56e59a